packages/cozy-client/src/types.js

```diff
@@ -380,4 +380,26 @@ import { QueryDefinition } from './queries/dsl'
  * @property {string} [codeChallenge]
  */
 
+/**
+ * Subdomain type for a Cozy. Can be flat or nested subdomains
+ *
+ * Example of 'flat' domain: https://claude-notes.somedomain.fr
+ * Example of 'nested' domain: https://notes.claude.somedomain.fr
+ *
+ * @typedef {'flat'|'nested'} SubdomainType
+ */
+
+/**
+ * Represents the differents parts of a deconstructed Cozy link
```

Comment:
```suggestion
 * Represents the different parts of a deconstructed Cozy link
```

Reply:
Fixed in https://github.com/cozy/cozy-client/compare/637711a2193c5077ee2ada1cb7eec31c843ee6fc..f14039dd1b109d01d61f354cbd854fdb7804edf8

Thanks